extensions/amp-sticky-ad/0.1/amp-sticky-ad.js

```diff
@@ -238,6 +238,6 @@ class AmpStickyAd extends AMP.BaseElement {
   }
 }
 
-AMP.extension('amp-sticky-ad', AMP => {
+AMP.extension('amp-sticky-ad:0.1', AMP => {
```

Comment:
Actually `amp-sticky-ad` is only used in its own tests and the `amp-ad-3p-impl` tests. For the ad test, we don't really care about stickyAd class, all we want is a container with name `amp-sticky-ad`